x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_table.tsx

```diff
@@ -56,7 +56,11 @@ export const EngineTable: ReactFC<IEngineTableProps> = ({
       name: i18n.translate('xpack.appSearch.enginesOverview.table.column.name', {
         defaultMessage: 'Name',
       }),
-      render: name => <EuiLink {...engineLinkProps(name)}>{name}</EuiLink>,
+      render: name => (
```

Comment:
```suggestion
      render: (name) => (
```

I think some Prettier settings recently changed, I had to do an eslint pass and it caught some things that needed changing